jsettlers.logic/src/main/java/jsettlers/ai/economy/BuildingListEconomyMinister.java

```diff
@@ -339,7 +348,7 @@ private void addMinimalBuildingMaterialBuildings() {
 			addIfPossible(STONECUTTER);
 			addIfPossible(LUMBERJACK);
 			addIfPossible(FORESTER);
-			buildingsToBuild.add(TOWER);
+			buildingsToBuild.add(MEDIUM_LIVINGHOUSE);
 			buildingsToBuild.add(MEDIUM_LIVINGHOUSE);
```

Comment:
Yes I am sure.
First of all with low goods you need more bearers at that time because almost every settler is consumed as a builder, digger or worker.  Tests showed that if you build only a small living house you will need another small living house soon because 10 more settlers are not enough and two small living houses are nearly the same costs like a medium one but for a medium you get 30 instead of 20 settlers. So the first medium living house is for the basic bearers.
The second living house is to occupy the necessary land for the first mines for the tool smith for more lumberjacks and stonecutters. With this second living house you get only up to 30 pioneers of the planned 60. But this is a good compromise between enough pioneers to get enough land fast and costs before the AI starts with the additional lumberjacks and stonecutters.